src/app/page.tsx

```diff
@@ -26,11 +27,20 @@ export default function Home() {
 
   const ctaButton = useMemo(() => {
     if (isClient) {
-      return isConnected ? (
+      return isConnected || true ? (
```

Comment:
wont this always return true?

Reply:
Ah yes, I put that in to force the other option to show up, will remove